src/components/Board/Board.component.js

```diff
@@ -104,7 +107,71 @@ export class Board extends Component {
 
     this.state = {
       openTitleDialog: false,
-      titleDialogValue: props.board && props.board.name ? props.board.name : ''
+      titleDialogValue: props.board && props.board.name ? props.board.name : '',
+      runLiveHelp: false,
+      runLiveUnlock: false,
+      hideCloseButton: true,
```

Comment:
`hideCloseButton` is a boolean, a better name would be `isCloseButtonHidden` ("hideCloseButton" is a confusing name for a boolean since it implies an action).

Reply:
`hideCloseButton` is a prop from Joyride Library. it shows or hides the close icon button in the corner of the modal. for defect, it's no hide.

Reply:
Yes, I think so too. 
<img width="384" alt="Screenshot 2021-05-28 at 10 16 45 AM" src="https://user-images.githubusercontent.com/56564891/119945211-d177eb80-bf9d-11eb-8124-8afc12a7b9f5.png">
https://docs.react-joyride.com/props